api/app.js

```diff
@@ -0,0 +1,25 @@
+const bodyParser = require('body-parser');
+require('dotenv').config();
+const routes = require('../api/routes/index');
```

Comment:
1. Abs path
2. If you are importing form folder, `require` make a lookup for index